.gitignore

```diff
@@ -17,9 +17,6 @@ logs
 *.log
 junit*
 
-# Core dumps
-core*
-
```

Comment:
> If one has a core directory that they want to include it can be added back with the ! and adding the directory path to it.

I guess I think, this is too much trouble for what it is, and it's going to cause people to write code and make commits, not commit all the files they worked on because of a git ignore issue, and then lose their work.

AFAIK no one has ever accidentally committed a core file to our repo. My guess is that it's because the executables are run from the target directory, and that directory is already gitignored. So ignoring core* seems like overkill.

There's also no reason to think that files starting with `core*` are actually core dumps, the core pattern can be anything and often starts with a time or a process id.